tests/Http/Controllers/Api/Geojson/ProjectImagesControllerTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Biigle\Tests\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use ApiTestCase;
+use Faker\Factory as Faker;
+use Biigle\{Label, Image, Annotation, AnnotationLabel};
+use GeoJson\GeoJson as GeoJson;
+use Biigle\Tests\ProjectTest;
+use Biigle\Tests\VolumeTest;
+use Biigle\Tests\LabelTest;
+use Biigle\Tests\ImageTest;
+use Biigle\Tests\AnnotationTest;
+use Biigle\Tests\AnnotationLabelTest;
+
+class ProjectImagesControllerTest extends ApiTestCase
+{
+  public function testIndex()
+  {
+    $faker = Faker::create();
+    $firstProject = $this->project();
+    $secondProject = ProjectTest::create();
+
+    $volume = $this->volume();
+    $volume_2 = VolumeTest::create();
+    $secondProject->volumes()->save($volume);
+
+
+    $label_names = ["sponge", "jellyfish", "starfish"];
+    $labels = collect($label_names)->map(function($name)
+    {
+      return LabelTest::create(["name"=>$name]);
+    });
+    for($i=0;$i<5;$i++)
+    {
+      $image = ImageTest::create(['volume_id' => $volume->id,
+                                  'filename'=>"$i.jpg",
+                                  'lat'=>$faker->latitude(-90,90),
+                                  'lng'=>$faker->longitude(-180, 180)]);
+      $annotation = AnnotationTest::create(['image_id' => $image->id]);
+      AnnotationLabelTest::create(['annotation_id' => $annotation->id, 'label_id' => $labels->random()->id]);
+    }
+
+    for($i=0;$i<5;$i++)
+    {
```

Comment:
Please use spaces and no linebreak before the curly brace for `for` loops. Linebreaks before curly braces should only be used for classes and class methods (i.e. not for inline functions, too). This applies to the rest of the code, too.

```suggestion
    for ($i = 0; $i < 5; $i++) {
```